docs/src/main/asciidoc/_admin-guide/Command_Line_Interface.adoc

```diff
@@ -116,6 +116,45 @@ formatted as a command help content:
 help /subsystem=elytron/key-store=?:add
 ----
 
+[[cli-keyboard-navigation]]
+== Keyboard navigation
+
+In order to efficiently edit commands, the CLI allows you to navigate the words
+ and characters of a command using the keyboard.
+
+NB: Part of this navigation is platform dependent. 
+
+=== Go left (back) one word
+
+* _Alt+B_ : Linux, Windows.
+* _Ctrl+LeftArrow_: Linux, Solaris, HP-UX.
+* _Alt+LeftArrow_: Mac OSX.
+
+=== Go right (forward) one word
+
+* _Alt+F_ : Linux, Windows.
```

Comment:
I tested Alt+B and Alt+F on Solaris and HP-UX and it worked for me as well.

Reply:
Ok, I was expecting this but was not able to test it with my keyboard+Mac+VirtualBox context. Will update doc.